src/DOMPoint.js

```diff
@@ -46,6 +36,11 @@ class DOMPointReadOnly {
     get z() { return _(this).z }
     get w() { return _(this).w }
 
+    set x(value) { _(this).x = Number(value) }
+    set y(value) { _(this).y = Number(value) }
+    set z(value) { _(this).z = Number(value) }
+    set w(value) { _(this).w = Number(value) }
+
```

Comment:
If we put these back in the other class, then we can force `DOMPointReadOnly` to be read-only even if someone extends from it.